doc/lua-filters.md

```diff
@@ -371,6 +371,56 @@ function Meta(m)
 end
 ```
 
+## Adding additional text
+
+This filter adds 'Hello World!' to the beginning of the document, and
+'Good Bye!' to the end of the document.
+
+
+```lua
+function Doc(doc)
+  table.insert(doc.blocks, 1, pandoc.Para{ pandoc.Str 'Hello World!' })
+  table.insert(doc.blocks, pandoc.Para{ pandoc.Str 'Good Bye!' })
+  return pandoc.Pandoc(doc.blocks, doc.meta)
+end
+```
+
+## Adding additional text to only specific output formats
+
+This filter adds 'Hello World!' to the beginning of the document, and
+'Good Bye!' to the end of the document, _only if_ the output format is
+`HTML`.
+
+```lua
+function Doc(doc)
+  table.insert(doc.blocks, 1, pandoc.RawBlock('html', "<p>Hello World</p>"))
+  table.insert(doc.blocks, pandoc.RawBlock('html', "<p>Good Bye!</p>"))
+  return pandoc.Pandoc(doc.blocks, doc.meta)
+end
+```
+
+## Adding a custom user-variable and additonal text to only specific output formats
+
+This filter adds 'Hello World!' to the beginning of the document, and
+'Good Bye!' to the end of the document, _only if_ the output format is
+`odt`.  
+
+Furthermore, it adds a custom user-variable called 'producer'.
+
+```lua
```

Comment:
I like this example best, although I'm not sure whether the use of `RawBlock` is aproriate here. If the intention is do demonstrate how to add text to the start and end of a document, a normal Para as in the first example above should do.

Note that the separate filter on Meta is not needed, as one can modify `doc.meta` in the `Pandoc` filter function.